test/elixir_analyzer/test_suite/basketball_website_test.exs

```diff
@@ -57,25 +57,6 @@ defmodule ElixirAnalyzer.ExerciseTest.BasketballWebsiteTest do
     end
   end
 
-  describe "explicit nil" do
-    test_exercise_analysis "there should be no explicit nil handling",
-      comments_include: [ElixirAnalyzer.Constants.basketball_website_no_explicit_nil()] do
-      defmodule BasketballWebsite do
-        def extract_from_path(data, path) do
-          paths = String.split(path, ".", trim: true)
-          do_extract(data, paths)
-        end
-
-        defp do_extract(nil, _), do: nil
-        defp do_extract(data, []), do: data
-
-        defp do_extract(data, [path | next]) do
-          do_extract(data[path], next)
-        end
-      end
-    end
-  end
-
   describe "using Map" do
     test_exercise_analysis "there should be no usage of the map module",
       comments_include: [ElixirAnalyzer.Constants.basketball_website_no_explicit_nil()] do
```

Comment:
Oops, I see we made a mistake in this test in the past. It shouldn't use the comment about nil. In fact, after this PR, the "constant" `ElixirAnalyzer.Constants.basketball_website_no_explicit_nil()` should be completely gone - can you remove it from the `Constants` module?
```suggestion
      comments_include: [ElixirAnalyzer.Constants.basketball_website_no_map()] do
```

Reply:
I have just amended by your feedback.